centraldogma/dogma.py

```diff
@@ -312,3 +314,23 @@ def listener(revision: Revision, value: MyType) -> None:
         )
         watcher.start()
         return watcher
+
+    def merge_files(
+        self,
+        project_name: str,
+        repo_name: str,
+        merge_sources: List[MergeSource],
+        json_paths: Optional[List[str]] = None,
+        revision: Optional[int] = None,
+    ) -> MergedEntry:
+        """
+        Returns the merged result of files represented by ``MergeSource``. Each ``MergeSource``
+        can be optional, indicating that no error should be thrown even if the path doesn't exist.
+        If ``json_paths`` is specified, each ``json_path`` is applied recursively on the merged
+        result. If any of the ``json_path``s is invalid, a ``QueryExecutionException`` is thrown.
+
```

Comment:
Could you add [`ValueError`](https://github.com/line/centraldogma-python/pull/34/files#diff-c0cbe9e6e370518d8302068abd5cc76c7b27c06d5352372f2ffdf5810f03a17eR154) here?
```suggestion

    :raises ValueError: ...
```

Reply:
Oh I missed this 😅  Thanks!